metaci/release/utils.py

```diff
@@ -36,6 +36,19 @@ def jwt_for_webhook():
     )
 
 
+def parse_change_case_link(change_case_link):
+    if (
+        change_case_link
+        and len(
+            change_case_link.split(f"{settings.METACI_CHANGE_CASE_URL_TEMPLATE[:-9]}")
+        )
+        > 1
+    ):  # 500 to parse beginning sequence of change case; indexing [:-9] to remove "{case_id}" from url link
+        return f"500{change_case_link.split(settings.METACI_CHANGE_CASE_URL_TEMPLATE[:-9])[1]}"
+    else:
+        raise Exception("Please provide a valid change case link.")
```

Comment:
This looks pretty complicated and too closely tied to expecting METACI_CHANGE_CASE_URL_TEMPLATE to be in a particular format. I think we should:
- only store a case id in the change_case_link field, instead of a full URL like now
- construct the full URL from the id using METACI_CHANGE_CASE_URL_TEMPLATE when we need to